docs/community/ecosystem.md

```diff
@@ -0,0 +1,72 @@
+
+
+## Highlighted Project
+
+[![Promo image](https://cdn-images-1.medium.com/max/800/1*PwIMdZM7tpt3rmcyhlk2FQ.png)](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+#### [Keras-MXNet v2.2 released!](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+
+
+## Contents
+
+* [Learning MXNet and other Educational Resources](#learning-mxnet-and-other-educational-resources)
+* [MXNet APIs](#mxnet-apis)
+* [Toolkits to Extend MXNet](#toolkits-to-extend-mxnet)
+* [Debugging and Visualization](#debugging-and-visualization)
+* [Model Serving](#model-serving)
+* [Model Zoos](#model-zoos)
+* [Contributions](#contributions)
+
+
+## Learning MXNet and other Educational Resources
+
+* [Gluon 60 Minute Crash Course](https://gluon-crash-course.mxnet.io/) - deep learning practitioners can learn Gluon quickly with these six 10-minute tutorials.
+    - [YouTube Series](https://www.youtube.com/playlist?list=PLkEvNnRk8uVmVKRDgznk3o3LxmjFRaW7s)
+* [The Straight Dope](https://gluon.mxnet.io/) - a series of notebooks designed to teach deep learning using the Gluon Python API for MXNet.
+
+
+## MXNet APIs
+
+* [Clojure API](https://github.com/apache/incubator-mxnet/tree/master/contrib/clojure-package) - use MXNet with Clojure.
+* [C++ API](../api/c++/index.html) - not be confused with the C++ backend, this API allows C++ programmers to train networks in C++.
+* [Gluon Python Interface](../gluon/index.html) - train complex models imperatively and then deploy with a symbolic graph.
+* [Julia API](../api/julia/index.html) *(Community Supported)* - train models with multiple GPUs using Julia.
+* [Keras-MXNet](https://github.com/awslabs/keras-apache-mxnet) - design with Keras2 and train with MXNet as the backend for 2x or more speed improvement.
+* [MinPy](https://github.com/dmlc/minpy) - Pure numpy practice with third party operator integration and MXNet as backend for GPU computing
+* [Module Python API](../api/python/index.html) - backed by the Symbol API, you can define your network in a declarative fashion.
+* [ONNX-MXnet API](../api/python/contrib/onnx.html) - train and use Open Neural Network eXchange (ONNX) model files.
+* [Perl API](../api/perl/index.html) *(Community Supported)* - train models with multiple GPUs using Perl.
+* [R API](https://mxnet.incubator.apache.org/api/r/index.html) *(Community Supported)* - train models with multiple GPUs using R.
+* [Scala Infer API](../api/scala/infer.html) - model loading and inference functionality.
+* [TensorFuse](https://github.com/dementrock/tensorfuse) - Common interface for Theano, CGT, TensorFlow, and MXNet (experimental) by [dementrock](https://github.com/dementrock)
+
+
+## Toolkits to Extend MXNet
+
+* [NVIDIA DALI](https://github.com/NVIDIA/DALI/tree/master/examples/mxnet) - a collection of highly optimized building blocks and an execution engine to accelerate input data pre-processing for deep learning applications.
```

Comment:
delete

Reply:
It's not specifically built for mxnet, even though it has examples for using in mxnet. Examples in the same line of argument is that we don't include numpy, skimage, PIL, or opencv here.

Reply:
Apps team requested it to be added.

Reply:
The code base of DALI doesn't depend on mxnet at all, and only its example does for the purpose of demo. In this sense DALI is no more of "Toolkits to Extend MXNet" than numpy, skimage, PIL, opencv. I'd prefer if we don't start calling the MXNet examples in any other repos "Toolkits to Extend MXNet".

Reply:
DALI is an I/O library that has MXNet plugin, so it's probably closer to being a dependency of MXNet.

Reply:
Ok, so would this be a better spot for it?
https://github.com/apache/incubator-mxnet/tree/master/example#tools-with-mxnet

Reply:
Yes, I think that should be appropriate.